md_api/src/main/java/com/wipro/ats/bdre/md/api/InitJob.java

```diff
@@ -224,12 +215,9 @@ public static InitJobInfo parseBean(List<InitJobRowInfo> initJobRowInfos) {
      * command line notations.
      * @return This method return ouptut of InitJob proc as an instance of class InitJobRowInfo.
      */
-    @Autowired
-    private JobDAO jobDAO;
 
     public List<InitJobRowInfo> execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)